include/libswiftnav/signal.h

```diff
@@ -71,12 +94,17 @@ int sid_to_string(char *s, int n, gnss_signal_t sid);
  */
 bool sid_valid(gnss_signal_t sid);
 
-/** Converts the global index i in [0, NUM_SATS) to the corresponding sid
+/** Converts the global index i in [0, NUM_CODES) to the corresponding sid
```

Comment:
What's the idea behind this, wouldn't this require also constellation and code as parameter?